src/utils/prisma.error.ts

```diff
@@ -0,0 +1,35 @@
+import { Prisma } from '@prisma/client';
+
+const logPrismaError = (e: unknown) => {
+  if (e instanceof Prisma.PrismaClientValidationError) {
+    console.log(
+      'Prisma client validation error. A field is missing or of the incorect type.',
+    );
+  } else if (e instanceof Prisma.PrismaClientKnownRequestError) {
+    console.log('Prisma error code: ', e.code);
+    switch (e.code) {
```

Comment:
Instead of a switch I also just use a if, since you just check for one case.

Reply:
Replaced "switch" with "if"